packages/ui-components/Common/Select/SelectScrollButton/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
+import { useEffect, useRef, useState } from 'react';
+import { type FC, type RefObject } from 'react';
+
+import styles from '@node-core/ui-components/Common/Select/index.module.css';
+
+type SelectScrollButtonProps = {
+  direction: 'up' | 'down';
+  selectContentRef?: RefObject<HTMLDivElement | null>;
+  scrollAmount?: number;
+  scrollInterval?: number;
+};
+
+const SelectScrollButton: FC<SelectScrollButtonProps> = ({
```

Comment:
Primitive radix select already has [ScrollUpButton](https://www.radix-ui.com/primitives/docs/components/select#scrollupbutton) and [ScrollDownButton](https://www.radix-ui.com/primitives/docs/components/select#scrolldownbutton) Instead of adding these features, you can use what radix provides directly. Here is an example of primitive select anatomy; https://www.radix-ui.com/primitives/docs/components/select#anatomy